src/js/account/components/API/APIKey.tsx

```diff
@@ -76,18 +76,20 @@ export default function APIKey({ apiKey }: APIKeyProps) {
                             name="permissions"
                         />
 
-                        <ButtonToolbar>
-                            <Button color="red" onClick={() => removeMutation.mutate({ keyId: apiKey.id })}>
-                                Delete
-                            </Button>
-                            <Button
-                                color="blue"
-                                disabled={isEqual(watch("permissions"), apiKey.permissions)}
-                                type="submit"
-                            >
-                                Update
-                            </Button>
-                        </ButtonToolbar>
+                        <div className="flex items-center justify-end mb-2.5">
+                            <div className="space-x-1.5">
```

Comment:
Use Flex [gap](https://tailwindcss.com/docs/gap) in parent component instead of `space`.